tests/integration/conftest.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Canonical Ltd.
+# Licensed under the GPLv3, see LICENCE file for details.
+
+import configparser
+import re
+
+import pytest
+import pytest_asyncio
+import pytest_operator.plugin
+
+
+@pytest.fixture
+def openstack_environment(request):
+    """Parse the openstack rc style configuration file from the --openstack-rc argument
+
+    Return a dictionary of environment variables and values.
+    """
+    rc_file = request.config.getoption("--openstack-rc")
+    with open(rc_file) as f:
```

Comment:
this is possible with a nice one liner: `rc_file = pathlib.Path(rc_file).read_text()`